src/Perlang.Interpreter/PerlangInterpreter.cs

```diff
@@ -1702,6 +1703,69 @@ public VoidObject VisitWhileStmt(Stmt.While stmt)
                     }
                     else
                     {
+                        // TODO: "Operands must be numbers" isn't completely correct here.
```

Comment:
...since the operands can be numbers, but of unsupported types. We'll live with this being slightly incorrect for now.